cpp/src/dataframe/column.cpp

```diff
@@ -24,7 +24,7 @@
 #include "utilities/error_utils.h"
 #include "rmm/rmm.h"
 #include <cuda_runtime_api.h>
-#include "type_dispatcher.hpp"
+#include "types/type_dispatcher.hpp"
```

Comment:
I really think this is a utility. It should be under utils.  We don't have a separate folder for every other type of utility in cuDF.